ui/raidboss/data/triggers/o8n.js

```diff
@@ -6,110 +6,401 @@
+        '--Reset--': '--Réinitialisation--',
+        '--sync--': '--synchronisation--',
+        '--targetable--': '--ciblable--',
+        '--untargetable--': '--impossible à cibler--',
+        'Enrage': 'Enragement',
+
+        'Aero Assault': 'Méga Vent Véhément',
+        'Blizzard Blitz': 'Méga Glace Glissante',
+        'Flagrant Fire': 'Méga Feu Faufilant',
+        'Graven Image': 'Statue Divine',
+        'Gravitas': 'Tir Gravitationnel',
+        'Holy Ascent': 'Ascension Sacrée',
+        'Hyperdrive': 'Colonne De Feu',
+        'Idyllic Will': 'Volonté Idyllique',
+        'Indolent Will': 'Volonté Indolente',
+        'Indomitable Will': 'Volonté Indomptable',
+        'Indulgent Will': 'Volonté Indulgente',
+        'Inexorable Will': 'Volonté Inexorable',
+        'Intemperate Will': 'Volonté Intempérante',
+        'Light Of Judgment': 'Triade Guerrière',
+        'Mana Charge': 'Concentration De Mana',
+        'Mana Release': 'Décharge De Mana',
+        'Pulse Wave': 'Pulsation Spirituelle',
+        'Revolting Ruin': 'Méga Ruine Ravageuse',
+        'Shockwave': 'Onde De Choc',
+        'Thrumming Thunder': 'Méga Foudre Fourmillante',
+        'Timely Teleport': 'Téléportation Turbulente',
+        'Ultima Upsurge': 'Ultima Ulcérante',
+        'Vitrophyre': 'Vitrophyre',
+        'Wave Cannon': 'Canon Plasma',
+
+        'Blizzard+Thunder': 'Méga Glace + Méga Foudre',
+        'Half Arena': "Moitié d'arène",
+        'Statue Gaze': 'Regard statue',
+
+        'All Things Ending': 'Fin De Toutes Choses',
+        'Blizzard III': 'Méga Glace',
+        'Celestriad': 'Tristella',
+        'Explosion': 'Explosion',
+        'Fire III': 'Méga Feu',
+        'Forsaken': 'Cataclysme',
+        "Future's End": "Fin Du Futur",
+        'Futures Numbered': 'Ruine Du Futur',
+        'Gravitational Wave': 'Onde Gravitationnelle',
+        'Heartless Angel': 'Ange Sans Cœur',
+        'Heartless Archangel': 'Archange Sans Cœur',
+        'Hyperdrive': 'Colonne De Feu',
+        'Idyllic Will': 'Volonté Idyllique',
+        'Indulgent Will': 'Volonté Indulgente',
+        'Light Of Judgment': 'Triade Guerrière',
+        'Meteor': 'Météore',
+        'Pasts Forgotten': 'Ruine Du Passé',
+        'Pulse Wave': 'Pulsation Spirituelle',
+        'Starstrafe': 'Fou Dansant',
+        'The Path Of Light': 'Voie De Lumière',
+        'Thunder III': 'Méga Foudre',
+        'Trine': 'Trine',
+        'Trine (big)': 'Trine (grand)',
+        'Trine (small)': 'Trine (petit)',
+        'Ultima': 'Ultima',
+        'Ultimate Embrace': 'Étreinte Fatidique',
+        'Wings Of Destruction': 'Aile De La Destruction',
+
+        'Soak': 'Absorber',
+        'Past/Future': 'Passé/Futur',
+        'Past/Future End': 'Passé/Fin du futur',
+        'Knockback Tethers': 'Liens de projection',
+        'Sleep/Confuse Tethers': 'Liens de Sommeil/Confusion',
+        'Statue Half Cleave': 'Demi clivage de la statue',
+      },
     },
-  ]
```

Comment:
You need to add back in this trailing `]` here.  This is what closes the `timelineReplace: []` block.

Reply:
Ah, that was a little accident. Gonna put it back in x)